third_party/render_md_test.go

```diff
@@ -0,0 +1,27 @@
+package devrel_tutorial
+
+import (
+	"testing"
+	"github.com/googlecodelabs/tools/claat/util"
+)
+
+func TestRenderHeadingMd(t *testing.T) {
+	h2 := &Heading{
+    Level: 2,
```

Comment:
Fix indentation here and below

Reply:
aha! good catch! I'm using Sublime text with same size tabs as two-spaces. Didn't catch this. This repo uses tabs for default spacing, for some reason.